src/imputors/knn.jl

```diff
@@ -34,40 +34,50 @@ function impute!(data::AbstractMatrix{Union{T, Missing}}, imp::KNN; dims=nothing
     X = d == 1 ? data : transpose(data)
 
     # Get mask array first
-    mmask = ismissing.(X)
+    Xmask = ismissing.(X)
```

Comment:
why not `X_mask`?
Isn't that more inline with BlueStyle?

Reply:
Also I wonder if we should call this `missing_mask`?
So i don't have to remember if it was a mask of missings or nonmissing?

Reply:
I didn't see anything explicit about variables names, just method names. I don't think this package follows BlueStyle anyways since this predates that guide. I can change it for now, but I think we should have a separate BlueStyle PR, to avoid inconsistencies within this package.

Reply:
I was considering using `missing_mask`, but that seemed a bit verbose when doing our indexing/slicing. I can just do the slicing on separate lines though 🤷